package.json

```diff
@@ -104,7 +104,8 @@
   "resolutions": {
     "eslint": "^7.1.0",
     "autoprefixer": "9.7.6",
-    "copy-to-clipboard": "3.2.0"
+    "copy-to-clipboard": "3.2.0",
+    "**/@fluentui/scripts/**/typescript": "4.1.3"
```

Comment:
The reason for this line is to force API Extractor to use TS 4.1.3 rather than 4.0.5. After MUCH fighting with yarn resolutions I finally figured out that you need the leading `**` to prevent hoisting from something under a monorepo package. 

Resolutions docs: https://classic.yarnpkg.com/en/docs/selective-version-resolutions/
Full spec: https://github.com/yarnpkg/rfcs/blob/master/implemented/0000-selective-versions-resolutions.md